docs/topics/feed-exports.rst

```diff
@@ -185,6 +185,27 @@ passed through the following settings:
  * :setting:`AWS_ACCESS_KEY_ID`
  * :setting:`AWS_SECRET_ACCESS_KEY`
 
+.. _topics-feed-storage-gcs:
+
+Google Cloud Storage (GCS)
+--------------------------
+
+The feeds are stored on `Google Cloud Storage`_.
+
+ * URI scheme: ``gs``
+ * Example URIs:
+
+   * ``gs://mybucket/path/to/export.csv``
+
+ * Required external libraries: `google-cloud-storage <https://cloud.google.com/storage/docs/reference/libraries#client-libraries-install-python>`_.
+
+For more information about authentication, please refer to `Google Cloud documentation <https://cloud.google.com/docs/authentication/production>`_.
+
+You can set a *Project ID* and *Access Control List (ACL)* through the following settings:
+
+ * :setting:`FEED_STORAGE_GCS_ACL`
```

Comment:
Since this setting is specific to Google Cloud storage, maybe the name should be simply `GCS_ACL`?

Reply:
Oh, I see this goes in line with #3607

Reply:
Yup.
Also, this aligns to other ACLs like FILE_STORAGE_ACL and so on.
My opinion is that we could refactor them, but to keep this PR shorter I prefer a different setting.

Reply:
I’ve [left a comment about this](https://github.com/scrapy/scrapy/pull/3607#discussion_r268109665) in #3607.

I would wait for that discussion to be over before merging these changes. I believe this pull request is ready to merge otherwise, though.

Reply:
Nevermind.